docs/public/deploying-a-devfile-using-odo-on-IBM-Z-and-Power.adoc

```diff
@@ -0,0 +1,93 @@
+
+* Development components such as container definition for build and application runtimes
+* A list of pre-defined commands that can be run
+* Projects to initially clone
+
+odo takes this devfile and uses it to create a workspace of multiple containers running on Kubernetes or OpenShift.
+
+Devfiles are YAML files with a defined https://devfile.github.io/devfile/_attachments/api-reference.html[schema].
+
+== odo and devfile
+
+odo can now create components from devfiles as recorded in registries. odo automatically consults the https://github.com/odo-devfiles/registry[default registry] but users can also add their own registries. Devfiles contribute new component types that users can pull to begin development immediately.
+
+An example deployment scenario:
+
+. `odo create` will consult the recorded devfile registries to offer the user a selection of available component types and pull down the associated `devfile.yaml` file
+. `odo push` parses and then deploys the component in the following order:
+ .. Parses and validates the YAML file
+ .. Deploys the development environment to your Kubernetes or OpenShift cluster
+ .. Synchronizes your source code to the containers
+ .. Executes any prerequisite commands
+
+== Deploying your first devfile on IBM Z & Power
+Since the DefaultDevfileRegistry does't support IBM Z & Power now, you will need to create a secure private DevfileRegistry first. To create a new secure private DevfileRegistry , please check the doc link:https://github.com/openshift/odo/blob/main/docs/public/secure-registry.adoc[secure registry].
+
+The images can be used for devfiles on IBM Z & Power
+[options="header"]
+|===
+|Language | Devfile Name | Description | Image Source | Supported Platform
+
+| Java
+| java-maven
+| Upstream Maven and OpenJDK 11
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8[codeready-workspaces/plugin-java11-openj9-rhel8]
+| s390x, ppc64le
+
+| Java
+| java-openliberty
+| Open Liberty microservice in Java
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8[codeready-workspaces/plugin-java11-openj9-rhel8]
+| s390x, ppc64le
+
+| Java
+| java-quarkus
+| Upstream Quarkus with Java+GraalVM
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java8-openj9-rhel8[codeready-workspaces/plugin-java8-openj9-rhel8]
+| s390x, ppc64le
+
+| Java
+| java-springboot
+| Spring Boot® using Java
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8[codeready-workspaces/plugin-java11-openj9-rhel8]
+| s390x, ppc64le
+
+| Vert.x Java
+| java-vertx
+| Upstream Vert.x using Java
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8[codeready-workspaces/plugin-java11-openj9-rhel8]
+| s390x, ppc64le
+
+| Node.JS
+| nodejs
+| Stack with NodeJS 12
+| link:https://registry.redhat.io/codeready-workspaces/plugin-java8-openj9-rhel8[codeready-workspaces/plugin-java8-openj9-rhel8]
```

Comment:
Clicking these links gives me not found. Which is ok as they are image urls but then they shouldn’t be urls as the first instinct of a user would be click the link.
Maybe point to the dockerfile of the image?